### PR TITLE
Add an entry about our screens

### PR DESCRIPTION
**Pull request by tijmenb**
This is based on [information from the old opsmanual][1]. It adds general info about our screen use, adds info about the search screen we maintain and adds some screenshots.

 👉 https://govuk-tech-docs-pr-90.herokuapp.com/opsmanual/screens.html

https://trello.com/c/dQNcnbd0

[1]: https://github.gds/pages/gds/opsmanual/2nd-line/screen.html

### CODE REVIEW

File: source/opsmanual/screens.html.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Most teams in GOV.UK have screens set up to show data about pull requests and
+releases.
+
+Most often displayed are the [deploy lag radiator][deploy-lag] and the [fourth wall](https://github.com/alphagov/fourth-wall).
+
+## Search screen
+
+![Screen shot of the search screen](images/search-screen.png)
+
+The [search screen][search-screen] displays live data from GOV.UK. Includes number of people on GOV.UK, latest searches, trending and recent content. It's publicly accessible because there's sometimes personal data in the latest searches.
```
Comment (36degrees) [inline source/opsmanual/screens.html.md]: Is there meant to be a negation in here somewhere? ("It's publicly accessible because")

Comment (issyl0) [inline source/opsmanual/screens.html.md]: I think you meant "it's *not* publicly accessible"?

Comment (36degrees) [inline source/opsmanual/screens.html.md]: Snaaaaap.

File: source/opsmanual/screens.html.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+### Icinga alert summary per environment
+
+This screen shows a summary of the critical and warning alerts for our
+three environments (production, staging, integration) in a colour coded
+box (red for criticals, yellow for warnings, green for all ok).
+
+This is powered by [blinken](https://github.com/alphagov/blinken), an
+instance of which is running on the localhost on port 8080. This is run
+by the govuk2ndline user by running `~/blinken/run.sh`.
```
Comment (issyl0) [inline source/opsmanual/screens.html.md]: Line 28 says "credentials [for the 2nd line screen] on the Wiki", but this line specifies one half of them. Is that intentional?

File: source/opsmanual/screens.html.md (hunk 1 above)
Comment (tijmenb) [inline source/opsmanual/screens.html.md]: That makes a lot more sense :)

File: source/opsmanual/screens.html.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+This is powered by [blinken](https://github.com/alphagov/blinken), an
+instance of which is running on the localhost on port 8080. This is run with `~/blinken/run.sh`.
+
+### Upcoming Releases
+
+This shows the list of upcoming releases.
+
+This is powered by the [GOV.UK departure
+lounge](http://govuk-departure-lounge.herokuapp.com) application, which
+reads from the release calendar.
```
Comment (issyl0) [inline source/opsmanual/screens.html.md]: Is it worth linking to the source for this, like we've done with `deploy-lag-radiator`?

File: source/opsmanual/screens.html.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+platform](https://grafana.publishing.service.gov.uk/#/dashboard/file/2ndline_health.json),
+a list of upcoming releases, and a dashboard showing the alerts for each
+environment
+
+This dashboard contains 2 graphs, one of origin 4xx and 5xx, and one of
+edge 4xx and 5xx. Its worth keeping an eye on this and looking for any
+anomalies, as this may indicate issues on production. Its likely due to
+our caching behaviour that the top graph of origin errors will indicate
+issues before they are visible in the 2nd graph, and to end users.
+
```
Comment (issyl0) [inline source/opsmanual/screens.html.md]: These two "its" need apostrophes. 🔎

File: source/opsmanual/screens.html.md (hunk 3 above)
Comment (tijmenb) [inline source/opsmanual/screens.html.md]: Is https://github.com/issyl0/departure-lounge the maintained fork? Should we move that to alphagov?

Comment (issyl0) [inline source/opsmanual/screens.html.md]: Yep, that's the maintained fork. I think I'd prefer to keep it on my GitHub (given it's also on my Heroku account, and naturally only I have permissions on that). People have raised PRs before and I'm generally pretty prompt on merging them (there are very rarely any changes anyway), but if people want semi-official things to live on alphagov, I'm open to that.

This does raise a bigger question about what we do with dashboards/20% projects (so we don't have single points of failure - should they all be under alphagov?), but that's probably not a discussion for here.

File: source/opsmanual/screens.html.md (hunk 1 above)
Comment (issyl0) [inline source/opsmanual/screens.html.md]: 🎉

**Pull request merged**